google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -73,3 +73,12 @@
         PRIMARY KEY (`id`)
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 """
+
+CREATE_GSUITE_ACCOUNT_GROUPS_TABLE = """
```

Comment:
a couple of nits here:
(1) can we avoid using "ACCOUNT" here?  and just leave this as CREATE_GSUITE_GROUPS_TABLE ?
(2) can we make this more generic, i.e not use "GSUITE"?  as to avoid any fallout from name changes in the future?  and also to make this more platform neutral?